ale_linters/eruby/erubi.vim

```diff
@@ -0,0 +1,33 @@
+" Author: Eddie Lebow https://github.com/elebow
+" Description: eruby checker using `erubi`
+
+function! ale_linters#eruby#erubi#GetExecutable(buffer) abort
+    call system('ruby -r erubi/capture_end -e ' . ale#Escape('""'))
```

Comment:
This system call blocks `vim`. There's a mechanism for handling these kinds of problems with more background jobs in ALE. See `:help ale-command-chain`, and you can see existing linters which use the `command_chain` option. You can return an empty string for any command callback for skipping them, including the last callback in the chain.